tests/storage/linstor/test_linstor_sr.py

```diff
@@ -78,6 +78,15 @@ def test_snapshot(self, vm_on_linstor_sr):
         finally:
             vm.shutdown(verify=True)
 
+    @pytest.mark.small_vm
+    @pytest.mark.big_vm
+    def test_resize_vdi(self, vm_on_linstor_sr):
+        vm = vm_on_linstor_sr
+        logging.info("* VDI Resize started *")
```

Comment:
Do we really want `*` marker in the logs?

Reply:
Removed it.